smac/intensification/successive_halving.py

```diff
@@ -351,23 +357,33 @@ def eval_challenger(self,
                                   "Interrupting current challenger and moving on to the next one")
                 # ignore all pending instances
                 self.curr_inst_idx = np.inf
+                status = StatusType.CAPPED
 
             # adding challengers to the list of evaluated challengers
-            self.curr_challengers.add(challenger)
+            #  - should not capped
+            #  - should be successful in at least 1 run
+            # curr_challengers is a set, so "at least 1" success can be counted by set addition
```

Comment:
Okay, so if something worked once, it's added to `curr_challengers`, and if it fails afterwards, it's in both? If yes, could you please clarify this in the comment?

Reply:
Yes. I'll add it